<h4>The Life of Brine</h4>

A $120$-gallon (gal) tank initially contains $90$ lb of salt dissolved in $90$ gal of water. Brine
containing $2$ lb/gal of salt flows into the tank at the rate of $4$ gal/min, and the well-stirred
mixture flows out of the tank at the rate of $3$ gal/min. How much salt does the tank contain
when it is full?

$$
x=2t+180+\frac{C}{(t+90)^{3}}
$$
solves
$$
\frac{dx}{dt}+\frac{3x}{t+90}=8
$$
Now plug in $x(0)=90$ and solve for $C$.

In [1]:
using Roots

In [2]:
f(C)=90+(C/(90^3))

f (generic function with 1 method)

In [6]:
D=find_zero(f,-10000000)

-6.561e7

In [7]:
2*30+180+(D/(120^3))

202.03125

The tank once full will contain about $202$ lb of salt.